python/src/pipeline/pipeline.py

```diff
@@ -3106,9 +3205,12 @@ def get_status_tree(root_pipeline_id):
         # the Datastore but were never run due to mid-flight task failures.
         child_pipeline_record = found_pipeline_dict.get(child_pipeline_key)
         if child_pipeline_record is None:
-          raise PipelineStatusError(
-              'Pipeline ID "%s" points to child ID "%s" which does not exist.'
-              % (pipeline_record.key().name(), child_pipeline_key.name()))
+          # If we limited the depth, we expect missing children, so don't warn.
+          if depth is None:
```

Comment:
The children may be missing due to eventual consistency. Also, I think this is a good case for best-effort display of the data given.

See https://github.com/Khan/appengine-pipelines/commit/87095b904f42d3788280bd1b0654887e8f8c7267